types/paypal__paypalhttp/lib/paypalhttp/http_client.d.ts

```diff
@@ -0,0 +1,48 @@
+import { IncomingHttpHeaders, ServerResponse } from 'http';
+import { Encoder } from './encoder';
+import { Environment } from './environment';
+import { FormEncoded } from './serializer/form_encoded';
+import { Json } from './serializer/json';
+import { Text } from './serializer/text';
+import { Multipart } from './serializer/multipart';
+
+export namespace HttpClient {
```

Comment:
do you need a namespace here? with moduel you can control what to export as type (e..g `HttpRequest`)

Reply:
Good idea. I will change it based on you comment